local-cli/generator/index.js

```diff
@@ -8,11 +8,41 @@
  */
 'use strict';
 
+var execSync = require('child_process').execSync;
 var fs = require('fs');
 var path = require('path');
+var semver = require('semver')
 var yeoman = require('yeoman-generator');
 var utils = require('../generator-utils');
 
+// Use Yarn if available, it's much faster than the npm client.
+// Return the version of yarn installed on the system, null if yarn is not available.
+function yarnVersionIfAvailable() {
+  let yarnVersion;
+  try {
+    if (process.platform.startsWith('win')) {
+      // TODO implement on Windows
```

Comment:
http://stackoverflow.com/questions/4507312/how-to-redirect-stderr-to-null-in-cmd-exe

Thanks to @kentaromiura 

What do you think of doing it in one go maybe?